doc/developer/documentationAraInstall.md

```diff
@@ -0,0 +1,67 @@
+

```

Comment:
General remark : Your PR add a lot of images but none has been add to this file, may I ask why ?

Reply:
I've inserted the images